src/include/distributed/commands.h

```diff
@@ -80,11 +80,11 @@ typedef enum DistOpsOperationType
  */
 typedef struct DistributeObjectOps
 {
-	char * (*deparse)(Node *);
```

Comment:
Can you revert only the last "indent" commit , lets not include these indent changes here.

Reply:
removed